manifests/i/IObit/AdvancedSystemCare/18.0.1.156/IObit.AdvancedSystemCare.installer.yaml

```diff
@@ -0,0 +1,32 @@
+# Automatically updated by the winget bot at 2024/Oct/11
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: IObit.AdvancedSystemCare
+PackageVersion: 18.0.1.156
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: Advanced SystemCare_is1
+ReleaseDate: 2024-08-26
+AppsAndFeaturesEntries:
+- DisplayName: Advanced SystemCare
+  Publisher: IObit
+  DisplayVersion: 17.6.0

```

Comment:
```suggestion
  DisplayVersion: 18.0.1
```

Reply:
@stephengillie

Reply:
Thanks - I hadn't got to fixing this one yet.